OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyEmailFragment.java

```diff
@@ -125,9 +125,6 @@ public void onClick(View v) {
         // initial values
         if (mAdditionalEmailModels == null) {
             mAdditionalEmailModels = new ArrayList<>();
-            if (mCreateKeyActivity.mAdditionalEmails != null) {
-                mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
-            }
```

Comment:
Please remove these changes from this pull request, they were already fixed in development branch.